docs/bigquery-usage.rst

```diff
@@ -232,11 +232,7 @@ Get rows from a table's data:
    >>> client = bigquery.Client()
    >>> dataset = client.dataset('dataset_name')
    >>> table = dataset.table(name='person_ages')
-   >>> rows, next_page_token = table.data(max_results=100)  # API request
-   >>> rows.csv.headers
```

Comment:
`fetch_data` does not rebuild table metadata (that would require an extra API call)..  `table.schema` is implemented as an accessor over `table._properties`, which is (re-)populated after `create`, `patch`, `update`, or `reload`.